crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -64,7 +55,7 @@ fn update_old_parents(world: &mut World, parent: Entity, children: &[Entity]) {
             });
         }
     }
-    push_events(world, moved);
+    world.send_event_batch(moved);
```

Comment:
I'm a bit confused. There is no alternative to the `World` when accessing data, as that's where all ECS data is stored. Everything already goes through the `World`.

This PR changes nothing in this regard. It just uses the helper methods defined on `World` instead of the one local to this module.

Reply:
Oh okay, I see what you are saying. This is just done to centralize the logic of where things are happening. Makes sense!